.github/workflows/update-docs.yml

```
@@ -0,0 +1,45 @@
+name: Update Docs
+on:
+  pull_request:
+
+jobs:
+  update_dfx_json_schema:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        rust: ["1.60.0"]
+        os: [ ubuntu-20.04 ]
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3.0.2
```
what do you think about using the latest?